executor/prepared.go

```diff
@@ -200,12 +203,9 @@ func (e *PrepareExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		vars.PreparedStmtNameToID[e.name] = e.ID
 	}
 
-	normalized, digest := parser.NormalizeDigest(prepared.Stmt.Text())
```

Comment:
why removing this?

Reply:
It seems that I made some mistakes using git. It's restored now.